commit f5b9e0724c1be704609cbe96ead47e4a7846aa24
Author: Matthias Reichl <hias@horus.com>
Date:   Sat Aug 25 20:21:40 2018 +0200

    rpi-wm8804-soundcard: use nicer driver_name "RPi-WM8804"
    
    The driver_name shows up in /proc/asound/devices and is used
    to load the proper Alsa card conf from /usr/share/alsa/cards.
    
    But the length is limited to 15 characters and spaces are
    replaced by underscore so the mangled name currently is
    "RPI_WM8804_soun" which is not very nice.
    
    Use "RPi-WM8804" instead which won't be mangled and can be used
    unaltered as the card conf filename and card name in that file.
    
    Signed-off-by: Matthias Reichl <hias@horus.com>

diff --git a/sound/soc/bcm/rpi-wm8804-soundcard.c b/sound/soc/bcm/rpi-wm8804-soundcard.c
index 50daaed73786..361d4784cbd7 100644
--- a/sound/soc/bcm/rpi-wm8804-soundcard.c
+++ b/sound/soc/bcm/rpi-wm8804-soundcard.c
@@ -313,7 +313,7 @@ static const struct of_device_id snd_rpi_wm8804_of_match[] = {
 };
 
 static struct snd_soc_card snd_rpi_wm8804 = {
-	.driver_name  = "RPI WM8804 soundcard",
+	.driver_name  = "RPi-WM8804",
 	.owner        = THIS_MODULE,
 	.dai_link     = NULL,
 	.num_links    = 1,
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1197_f5b9e0724c1be704609cbe96ead47e4a7846aa24.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1197_f5b9e0724c1be704609cbe96ead47e4a7846aa24.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
